### DISCUSSION TITLE
Sync Multiple Servers.

### DISCUSSION THREAD

**Discussion by DoubleAmperSand** [Taxel/PlexTraktSync#858] [Q&A] [answered]
Hi 
Would there be any issues with running two separate instances both syncing to the same Trakt login ?
Example (Main Home and Holiday Home - Not so good internet connection for streaming so need to play media locally) 
Unraid Server 1 with Plex in Location A 
Unraid Server 2 with Plex in Location B
Both have the PlexTraktSync docker running linked to the same Trakt account. 
What I'm trying to achieve is when I'm watching TV series at home and then travel to my holiday home the watched history from home would have synced to the my holiday home.

**Comment by glensc** [comment, marked as answer]
I think:

No technical limitations, only logical ones if they start to fight against each other. But you can turn off those sync options.

For example, marking as collected you should turn probably off, but syncing watched should be ok as there's no such sync option as marking as not watched.

Ou, wait, there's no Remove from Collection syncing, unless you are using "watch" subcommand.

It's probably best just experiment and use `--dry-run` option to see what would be synced.
